lua/alex/lazy/lsp.lua

```
@@ -17,55 +17,7 @@ return {
-              "SPL",
-              "standard",
-              "tokenizer",
-              "xml",
-              "xmlreader",
-              "xmlwriter",
-              "zip",
-              "zlib",
-              "wordpress",
-              "php"
-            },
+            stubs = { "bcmath", "bz2", "calendar", "Core", "ctype", "curl", "date", "dom", "exif", "fileinfo", "filter", "ftp", "gd", "gettext", "hash", "iconv", "imagick", "intl", "json", "libxml", "mbstring", "mysqli", "mysqlnd", "openssl", "password", "pcre", "PDO", "pdo_mysql", "Phar", "readline", "redis", "Reflection", "session", "SimpleXML", "soap", "sockets", "sodium", "SPL", "standard", "tokenizer", "xml", "xmlreader", "xmlwriter", "zip", "zlib", "wordpress", "php" },
```

Luacheck (1.2.0)

[warning] line is too long (480 > 120)

(W631)